src/style.css

```diff
@@ -1 +1,98 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, you could modify the `font-family` property to `font-family: Avenir-Heavy, Arial, sans-serif;` to ensure there is a fallback to a common font.